nixos/tests/tinydns.nix

```diff
@@ -1,26 +1,71 @@
-import ./make-test-python.nix ({ lib, ...} : {
+import ./make-test.nix ({ pkgs, lib, ... }:
```

Comment:
Please note that Perl tests [will be removed in NixOS 20.09](https://github.com/flokli/nixpkgs/commit/503dfbddf1baea81228b504682a91506eaa4ccda).

Reply:
Okay, I'll address that. Wasn't the case, when I first submitted this PR in Feb 2019. :)